app/assets/stylesheets/home.scss

```diff
@@ -1,9 +1,41 @@
 body {
-  background: image_url("polygon_green.jpg") no-repeat center center fixed;
+  background: image_url("gray_gradient.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
   background-size: cover;
-  background-position: center top;
-  overflow-x: hidden;
-  color: white;
-  font-family: Bandung, Arial, sans-serif;
-  font-weight: 400;
+}
+
+.footer {
+  background-color: #34495e;
```

Comment:
Color literals like `#34495e` should only be used in variable declarations; they should be referred to via variable everywhere else.